recipes/blacktex/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "blacktex" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/blacktex-{{ version }}.tar.gz
+  sha256: 1a0660ea8438ae7c640977a19b5b2c326a3555d0af2e32eb06e67758a4f0d40a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - importlib-metadata  # [py<38]
+    - python
+
+test:
+  imports:
+    - blacktex
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/nschloe/blacktex
+  summary: Cleans up your LaTeX files
+  license: GPL-3.0-only
+  license_file: LICENSE
+  description: |
+    ``blacktex`` is a little tool, helping with the article editing 
+    for LaTeX. It removes all comments from a given file and corrects 
+    some common anti-patterns.   
+  doc_url: https://github.com/nschloe/blacktex
+  dev_url: https://github.com/nschloe/blacktex
+
+extra:
+  recipe-maintainers:
+    - sugatoray
+
+# NOTE:
+#
+# Using SPDX IDs for about:license
+# - https://spdx.dev/ids/
+# - https://spdx.org/licenses/
+# - https://spdx.org/licenses/GPL-3.0-only.html
```

Comment:
SPDX identifiers have to be used for every recipe and are mentioned in the docs so it's not necessary to have it here.
```suggestion
```

Reply:
Done!